app/build.gradle

```diff
@@ -30,36 +28,26 @@ android {
 // Individual Google Play Services API : https://developers.google.com/android/guides/setup#add_google_play_services_to_your_project
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
-
```

Comment:
Could you not remove these newlines, I actually like spacing out dependencies based on function